lgc/patch/PatchInOutImportExport.cpp

```diff
@@ -299,6 +349,31 @@ void PatchInOutImportExport::processShader() {
   }
 }
 
+// =====================================================================================================================
+// Visits all "call" instructions against the callee functions in current entry-point function.
+
+// @param calleeFuncs : a list of candidate callee functions to check
+void PatchInOutImportExport::visitCallInstrs(ArrayRef<Function *> calleeFuncs) {
+  for (auto callee : calleeFuncs) {
+    for (auto user : callee->users()) {
+      if (CallInst *callInst = dyn_cast<CallInst>(user)) {
+        if (callInst->getFunction() == m_entryPoint)
+          visitCallInst(*callInst);
+      }
+    }
+  }
```

Comment:
nit: in the llvm coding style, you wouldn't need braces here

Reply:
That's not clear to me (I find that part of the LLVM standard really confusing). Anyway LLPC has its own rules: https://github.com/GPUOpen-Drivers/llpc/blob/dev/docs/CodingStandards.md#braces
And clang-format should be able to enforce the LLPC rules.

Reply:
Ack. I don't think clang-format can introduce/remove braces and only handles placement, doesn't it?

Reply:
Yep, seems like clang-tidy can do that but clang-format can't. Here's the set of rules used in the Tim's refactoring script:
https://github.com/GPUOpen-Drivers/llpc/blob/9c325414e55680666b64d783fba5df51ef559b3f/script/switch-coding-style.sh#L243
and I think it's consistent with your interpretation, @jayfoad. Thanks.

Reply:
Thanks for the link @jayfoad I missed that. Different people hold different opinion on this because either way has its own advantage, I will fix them now that it is explicitly documented.

Reply:
After reading the llvm coding style, I think I need to keep the braces if I read it correctly.
https://llvm.org/docs/CodingStandards.html#don-t-use-braces-on-simple-single-statement-bodies-of-if-else-loop-statements
> Use braces on the outer block because of a nested `if`
so the outer if needs braces here, and the outer for-loops also need braces because there are more than two levels of nesting.